internal/model/board.go

```diff
@@ -204,14 +295,59 @@ func NewBoard(size int) (*Board, error) {
 
 	board := &Board{
 		Tiles: make([][]*BoardTile, size),
+		Players: []*Player{
+			{
+				Color: ColorBlue,
+				Line:  0,
+				Row:   0,
+			},
+		},
+		State: GameStatePlaceTile,
 	}
 
-	for i := 0; i < size; i++ {
-		board.Tiles[i] = make([]*BoardTile, size)
-		for j := 0; j < size; j++ {
-			board.Tiles[i][j] = &BoardTile{
-				Tile:     tiles[(size*i)+j],
-				Rotation: randomRotation(),
+	tileIndex := 0
+	for line := 0; line < size; line++ {
```

Comment:
Maybe a comment to explain why you do this

Reply:
Done